docs/model_pipeline.rst

```diff
@@ -70,13 +72,23 @@ Rules
 
    prepare_resources/*
 
+
 Outputs
 -------
 
 Output files are saved in :file:`results/_resources`.
 
 The resources are preprocessed data that serve as material for building scenarios.
-Because they are a first intermediate result, resources will be saved in :file:`results/_resources`.
+They are a first intermediate result in oemof-B3.
+
+File storage
+^^^^^^^^^^^^
+
+Resources are saved in :file:`results/_resources`.
```

Comment:
This duplicates the information above.

Reply:
Fixed with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/dd9d7487308e0ead4518342d310cbec0e9440127.